src/containers/ValetReservation.jsx

```diff
@@ -62,6 +86,11 @@ const ValetReservation = () => {
       <button onClick={() => handleGetQrCode(event)}>Show QR Code </button>
       {showQrCode && qrCodeLink ? <img src={qrCodeLink} /> : null}
       <Link to="/scanner">Scanner</Link>
```

Comment:
can delete the xmlns link for future refactor. I guess so we are all aligned for icons.

Reply:
Yup I'm leaving it in until we come together for the CSS refactor